crates/test/src/lib.rs

```diff
@@ -735,26 +818,84 @@ impl Runner<'_> {
 
     /// Executes a single test case.
     ///
-    /// Composes `runner_wasm` with `test_wasm` and then executes it with the
-    /// runner specified in CLIflags.
+    /// Composes `runner_wasm` with the components in `test_components` and then
+    /// executes it with the runner specified in CLI flags.
     fn runtime_test(
         &self,
         case: &Test,
         runner: &Component,
         runner_wasm: &Path,
-        test: &Component,
-        test_wasm: &Path,
+        test_components: &[(&Component, &Path)],
     ) -> Result<()> {
-        let mut config = wasm_compose::config::Config::default();
-        config.definitions = vec![test_wasm.to_path_buf()];
-        let composed = wasm_compose::composer::ComponentComposer::new(runner_wasm, &config)
-            .compose()
-            .with_context(|| format!("failed to compose {runner_wasm:?} with {test_wasm:?}"))?;
+        let document = match &case.config.wac {
+            Some(path) => {
+                let wac_config = case.path.join(path);
+                fs::read_to_string(&wac_config)
+                    .with_context(|| format!("failed to read {wac_config:?}"))?
+            }
+            // Default wac script is to just make the `test:test` component
+            // available to the `runner`.
+            None => "
+                package example:composition;
```

Comment:
will this default wac script work if you rename it with 

```//@ runner = "other-runner"``` 

as described in the docs above `Test Configuration: World Names`?

Reply:
Good point, I've updated this to account for custom names now too